build.gradle

```diff
@@ -79,7 +79,17 @@ sourceSets {
 	}
 }
 
+task composeDeploy(type: Exec) {
```

Comment:
I think this would be more of a `iTestSetup()` (with `iTestTearDown`) because it could be more than just a compose up.

Reply:
It can totally be renamed. The beauty of the thing is that it is now completely independent from the `jenkins-library` which only needs to be aware of the `itest` task.

Reply:
That is, indeed, very neat!

Reply:
Renaming done, tests on going